config/config.go

```diff
@@ -590,6 +590,12 @@ func (c *Config) Distro() string {
 	return K8SDistro
 }
 
+func (c *Config) IsVirtualSchedulerEnabled() bool {
+	k8sDistro := c.ControlPlane.Distro.K8S
+	return k8sDistro.Enabled && k8sDistro.Scheduler.Enabled ||
```

Comment:
Values.controlPlane.distro.k8s.enabled is false by default but implicitly enabled if no other distro is enabled

Reply:
Fixed it.